<?xml version='1.0' ?>

<launch>
  <node pkg="charger_server_ros2" exec="charger_service_server" name="charger_service">
    <param name="PLC_IP_address" value="192.168.1.1"/>
    <param name="PLC_Port_address" type="int" value="8501"/>
    <param name="timeout" type="float" value="10.0"/>
    <param name="frequency" type="float" value="2.0"/>
  </node>

  <node pkg="charger_fleet_client_ros2" exec="charger_fleet_client_ros2" name="fake_client_node" output="both">
    <param name="fleet_name" value="amr_vdm"/>
    <param name="charger_name" value="charger001"/>
    <!-- ros topic -->
    <param name="charger_state_topic" value="/charger_state"/>
    <param name="charging_trigger_server_name" value="/charger_server"/>
    <!-- dds topic -->
    <param name="dds_domain" value="52"/>
    <param name="dds_state_topic" value="charger_state"/>
    <param name="dds_charger_request_topic" value="charger_request"/>
    <param name="update_frequency" value="5.0"/>
    <param name="publish_frequency" value="1.0"/>
  </node>

</launch>
